src/foam/nanos/fs/FileType.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * @license
+ * Copyright 2020 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.nanos.fs',
+  name: 'FileType',
+  documentation: 'type of file such as PDF, JPGE, CSV',
+
+  properties: [
+    {
+      class: 'Long',
+      name: 'id',
+      visibility: 'RO'
+    },
+    {
+      class: 'String',
+      name: 'fileName',
```

Comment:
what is 'fileName' - can it just be 'name'? 
and 'id' should be string and GUID.  or is name and mimeType a unique combination? if so then the id of this model can be 
ids['name', 'mimeType'] 
And this is really the model for MimeType
with properties: 
type  and
subType
is it not?

Reply:
you are right, gona rewrite this